docs/use-github-in-bcgov/transferring-repos-to-bc-gov-orgs.md

```diff
@@ -0,0 +1,102 @@
+# Transferring repositories
+
+This page describes how to transfer repositories from a team or ministry's GitHub organization to either the [bcgov](https://github.com/bcgov) or [bcgov-c](https://github.com/bcgov-c) GitHub organization.
```

Comment:
Perhaps: 

This page explains the process to move repositories from a team or ministry's GitHub organization to either [bcgov](https://github.com/bcgov) or [bcgov-c](https://github.com/bcgov-c) GitHub organization.